classes/mender-sdimg.bbclass

```diff
@@ -149,7 +149,7 @@ IMAGE_CMD_sdimg() {
     ROOTFS_SIZE=$(expr $REMAINING_SIZE / 2)
 
     # create rootfs
-    dd if=/dev/zero of=${WORKDIR}/rootfs.$FSTYPE count=1 seek=$ROOTFS_SIZE bs=1M
+    dd if=/dev/zero of=${WORKDIR}/rootfs.$FSTYPE count=0 seek=$ROOTFS_SIZE bs=1M
```

Comment:
Yes, the other py.test run the tests using qemu, which mounts the image being made here. I'll double check manually just to be sure...